### ISSUE TITLE
Roll out internal study on user flow and artifacts

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#59001]

**Comment by sterkenburgsara** [comment]
@allison0034 Initial plan draft in [Google doc here](https://docs.google.com/document/d/1He1MzKuIokocU4TC9K2NaNS7sEHvtepjr_rR3yiedOc/edit?usp=sharing) - curious to get your thoughts on whether you think backup on-support person would be a good fit for this & how long this effort should take place. Please comment in there if you have different ideas - I would value your suggestions. Thank you! // cc @shiragoodman

**Comment by sterkenburgsara** [comment]
@allison0034 good point! I think we should also have a question about whether or not entry/exit points were accounted for. I'll put message in Slack governance channel to vote on more straightforward things and then thread a message re: questions to ask in survey once that concludes. Thank you!

**Comment by sterkenburgsara** [comment]
Update: On 6/6/23 the governance team members voted ([Slack thread](https://dsva.slack.com/archives/GNF5GSD9N/p1686079662639229)) on the following approaches to a survey to track data on user flows: 
* Use a Google form 
* The backup on-support person each week will fill out the survey following Design Intent touchpoints
* The survey will run for 10 design intents before closing. 

The following questions will be included in the survey:
* Did the team create a user flow? (Y/N)
* Did the flow account for more than just the happy path? (Y/N)
* Did the flow account for all user scenarios?
* Does the user flow have swim lanes? (Y/N)
* Does it show the intended entry & exit points? (Y/N)
* Was the user flow easy to understand? (Y/N)
* Are there any other comments or notes about the user flow? (open-ended)

**Comment by sterkenburgsara** [comment]
Update: 6/6/23 - this ticket is in progress and is not yet complete; it will move into the next sprint. Remaining tasks include: building survey in a Google form and modifying moderator responsibility documents for Design Intent.

**Comment by sterkenburgsara** [comment]
@allison0034 [draft survey in Google Forms](https://docs.google.com/forms/d/e/1FAIpQLScso06ymYlaSj6N2yuXxWdUocsDI6PGM-PwFqseYHX75WjqTw/viewform) ready for your review.

**Comment by allison0034** [comment]
Looks good to me!

**Comment by sterkenburgsara** [comment]
@allison0034 Great, thanks. Also [added a new (temporary) step 6](https://vfs.atlassian.net/wiki/spaces/PPS/pages/1748336706/Design+Intent+-+Moderator+Responsibilities) to the moderator responsibilities for Design Intent. This would need to be removed once the survey is taken 10 times. Any feedback on that? // cc @shiragoodman

**Comment by shiragoodman** [comment]
@sterkenburgsara looks good to me. and great idea having the back-up person be the person responsible for completing the survey. The only note I have is that the backup person should feel free to contact the rest of Governance team if they're unable to answer all questions. Not sure if that needs to be said, or if the team would just assume.

Other than that, no concerns! Thank you for putting this together!
